src/holon/models/config.py

```diff
@@ -99,6 +99,7 @@ def __str__(self) -> str:
 class QueryCovertModuleType(models.TextChoices):
     UPSCALING = "upscaling"
     COST = "cost"
+    COSTBENIFIT = "costbenifit"
```

Comment:
Is dit zo geschreven omdat Anylogic het zo verwacht, of mag dit "costbenefit" zijn?

Reply:
Hahaha oeps, goed dat je het gezien hebt. Dat moet zeer zeker gewoon costbenefit zijn. Pas het meteen aan!